workers/common/src/main/kotlin/common/env/ConfigFileBuilder.kt

```diff
@@ -77,6 +82,42 @@ class ConfigFileBuilder(val context: WorkerContext) {
             }
         }
 
+        /**
+         * Execute the given [block] to generate a proxy configuration if necessary.
+         * This variant is based on system properties that define https(s) proxy settings rather than
+         * environment variables. It is typically used by tools that have a strong connection to the Java ecosystem.
+         * If one of httpProxyHost or httpsProxyHost is defined, the [block] is invoked with a corresponding
+         * [ProxiesFromSystemProperties].
+         *
+         * @param block A lambda function that is an extension function on PrintWriter and takes a
```

Comment:
Nit: In the codebase, typically no `@param` tags are used, but the Kotlin convention to mention parameter names in the text of the comment.

Reply:
Fixed.